### PR TITLE
Oncoprint remove events

### LINKED ISSUE #215: OncoPrint component contains references to (deprecated) events
https://github.com/plotly/dash-bio/blob/d628310e143ebaaec2b0cd7cda49bd4cbc511cc5/src/lib/components/OncoPrint.react.js#L38

Doesn't affect functionality, but I think it's a bad idea to keep `fireEvent` anywhere in the component since events are deprecated.

### PR DESCRIPTION
**Pull request by shammamah**
Closes #215 

## About 
- [ ] This is a new component 
- [ ] I am adding a feature to an existing component, or improving an existing feature
- [x] I am closing an issue 

## Description of changes 
Remove references to Dash events, regenerate bundle, and release `0.0.8-rc4` on PyPI.

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
 ## [0.0.8]
 
+### Removed
+* Removed mentions of Dash events in OncoPrint component.
+
 ### Added
 * Added ability to define custom colours in styles parser for Molecule3D. 
 
+
```
Comment (mkcor) [inline CHANGELOG.md]: I guess we don't want this extra blank line.

File: dash_bio/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-bio",
-  "version": "0.0.8rc3",
+  "version": "0.0.8-rc5",
```
Comment (mkcor) [inline dash_bio/package.json]: Are we changing this notation based on a convention? If so, which one?

Comment (shammamah) [reply dash_bio/package.json]: We'll be using the dash going forward. I don't think it's particularly important.

File: CHANGELOG.md (hunk 1 above)
Comment (shammamah) [reply CHANGELOG.md]: Good catch!

File: dash_bio/package.json (hunk 2 above)
Comment (shammamah) [reply dash_bio/package.json]: (Just to be clear, the TestPyPI package removes the dash from the version name regardless!)

Comment (mkcor) [reply dash_bio/package.json]: I don't think either, I just like to have a reference (link) to other instances or discussions... That's how you build an ecosystem.

Comment (mkcor) [reply dash_bio/package.json]: (Yes, this I gathered, that's why I wanted to see where the inspiration came from, for adding the dash in the version number we display.)

Comment (mkcor) [reply dash_bio/package.json]: There I go with a reference (convention for hyphenating release candidate version notation): https://semver.org/spec/v2.0.0-rc.2.html

**Pull request merged**